### PR TITLE
feat(bajour/article-website): use teaser slider for article list & automatic iframe sizing

### PR DESCRIPTION
**Pull request by Itrulia**
Changed ArticleList to use the TeaserGrid implementation
Fixed teasers using css instead of styled

### CODE REVIEW

File: libs/block-content/website/src/lib/teaser/teaser.tsx
Comment (Itrulia) [inline libs/block-content/website/src/lib/teaser/teaser.tsx:1]: 1. change hover to use :hover in CSS. 
2. fixed that there's a preview image wrapper that is near identical to the image wrapper
3. fixed teaser title a11y & styling
4. fixed teaser lead styling

File: libs/block-content/website/src/lib/embed/embed-block.tsx
Comment (Itrulia) [inline libs/block-content/website/src/lib/embed/embed-block.tsx:1]: Came from another PR I merged into this one

File: apps/babanews/pages/index.tsx [hunk 1]
```diff
@@ -1,37 +1,10 @@
...
+import {styled} from '@mui/material'
+import {ApiV1, ContentWidthProvider, PageContainer} from '@wepublish/website'
 import {GetStaticProps} from 'next'
 import getConfig from 'next/config'
 
 const Frontpage = styled(PageContainer)`
-  gap: ${({theme}) => theme.spacing(6)};
   padding-bottom: ${({theme}) => theme.spacing(6)};
-
-  ${TeaserGridFlexBlockWrapper}:first-of-type ${TeaserWrapper}:first-of-type {
```
Comment (antkiewiczk) [inline apps/babanews/pages/index.tsx:17]: 👍

File: apps/bajour/src/components/website-builder-overwrites/blocks/teaser-overwrite.style.tsx [hunk 2]
```diff
@@ -37,11 +36,13 @@ export const TeaserImgStyled = styled(Image)`
...
-export const TeaserContentStyled = styled(TeaserContent)``
+export const TeaserContentStyled = styled(TeaserInnerWrapper)`
+  grid-column: initial;
+`
 
-export const TeaserPreTitleStyled = styled(TeaserPreTitle)``
+export const TeaserPreTitleStyled = styled('span')``
 
-export const TeaserTitlesStyled = styled(H3)`
+export const TeaserTitlesStyled = styled('h1')`
```
Comment (antkiewiczk) [inline apps/bajour/src/components/website-builder-overwrites/blocks/teaser-overwrite.style.tsx:45]: do you think that h1 is the best choice? shouldn't there be one h1 per page ideally?

File: libs/block-content/website/src/lib/teaser/teaser.tsx [hunk 3]
```diff
@@ -206,123 +212,75 @@ const teaserLinkStyles = () => css`
     'authors';
 `
 
-const teaserHeaderStyles = (theme: Theme) => css`
+export const TeaserTitle = styled('h1')`
```
Comment (antkiewiczk) [inline libs/block-content/website/src/lib/teaser/teaser.tsx:215]: same as above, do we want to use h1?

**Review by antkiewiczk — APPROVED** [review]

**Pull request merged**